src/hooks/multicall.ts

```diff
@@ -1,4 +1,7 @@
+import { Contract } from '@ethersproject/contracts'
+import { CallState as CallStateBase } from '@uniswap/redux-multicall'
 import { useWeb3React } from '@web3-react/core'
+import { Interface } from 'ethers/lib/utils'
```

Comment:
This should be @ethersproject/contract-interfaces or something similar, _not_ from the ethers umbrella package